emv2/org.osate.aadl2.errormodel.propagationgraph/src/org/osate/aadl2/errormodel/PropagationGraph/util/ExecuteResoluteUtil.java

```diff
@@ -108,9 +106,9 @@ private void addToSet(Map<String, SortedSet<NamedElement>> sets, String name, Na
 	}
 
 	public ExecuteResoluteUtil() {
-		Injector injector = ResoluteActivator.getInstance()
-				.getInjector(ResoluteActivator.COM_ROCKWELLCOLLINS_ATC_RESOLUTE_RESOLUTE);
-		injector.injectMembers(this);
+//		Injector injector = ResoluteActivator.getInstance()
+//				.getInjector(ResoluteActivator.COM_ROCKWELLCOLLINS_ATC_RESOLUTE_RESOLUTE);
+//		injector.injectMembers(this);
 	}
 
 	@Inject
```

Comment:
How does this get injected?

Reply:
removed all the gscope code in assure. Added injector back in in EMV2.